src/base/server_type.cpp

```diff
@@ -252,6 +262,20 @@ bool is_tran_server_with_remote_storage ()
   return false;
 }
 
+active_tran_server *get_active_tran_server_ptr ()
+{
+  if (is_active_transaction_server ())
+    {
+      assert (ats_Gl != nullptr);
+      return ats_Gl;
+    }
+  else
+    {
+      assert (is_passive_transaction_server ());
```

Comment:
How about assert(0)?

Reply:
I followed `get_passive_tran_server_ptr`. I guess this interface allows it to be called from any.

Reply:
Others(PTS, PS) except for the ATS never create(construct) the ats_Gl. I thinks it means that they don't access this. Can we need to open this function to them?

Reply:
Me neither. I'd have done just like `assert (is_active_transaction_server)` without any branch. However, I just respect and follow the existing function 'cause it's not the scope I think. If you'd like to change it, how about asking critsti, who wrote the code?

Reply:
@hornetmj applied with https://github.com/CUBRID/cubrid/pull/3867/files#r994312453.

Reply:
@cristiarg Please check this conversation.